### ISSUE TITLE
Can't watch replays of matches with a different computer level than what you currently have

### ISSUE THREAD

**Issue by Adam13531** [Adam13531/BotLand#42]
**Steps to reproduce**
The easiest way to repro this is to make a new account, attack someone else to produce a replay, then try to watch that replay from an account that has a different computer level.

**Expected outcome**
You can watch the replay (ignoring any authorization issues at least about how it may not be your replay).

**Actual outcome**
This error stack
```
TypeError: Cannot read property 'entitiesOnTile' of null
    at Arena.addEntityToTile
    at Arena.addEntity
    at Arena.createEntity
    at PubSub.publish
    at CliServRouter.route
    at Arena.playJournalEntry
    at Arena.setUpArenaBasedOnJournal
    at Main.jumpToSpecificReplayPhase
    at Main.setUpArenaForWatchingReplay
From previous event:
    at Main.onWatchReplay
    at <anonymous>:1:11
```

**Comment by Adam13531** [comment]
Note to self: this actually has to do with the width/height of the original Arena, and computer level is just a way that the width and height changes.

HiDeoo ran into this bug back before I modified computer levels to have different dimensions, so I believe the replay causing this had a 19x13 Arena. I then changed all Arenas lower than level 5 or 6 to be smaller than that, so entities couldn't be placed.
